src/main/pages/che-6/setup-openshift/openshift-config.adoc

```diff
@@ -66,35 +66,13 @@ CERTIFICATE=$(cat /path/to/openshift/ca.crt)
 oc new-app -f deploy/openshift/templates/multi/openshift-certificate-secret.yaml -p CERTIFICATE="${CERTIFICATE}"
 ----
 
-Once created, `OPENSHIFT_IDENTITY_PROVIDER_CERTIFICATE` env takes cert file content as a value and is then used in entrypoints of Che server and Keycloak images.
-
-image::workspaces/ca_secret.png[]
-
-* Deploy Che in https mode: link:openshift-single-user.html#https-mode[single user], link:openshift-multi-user.html#openshift-container-platform[multi-user]
-* Build an image with your certificate and push it to a registry. Example of a Dockerfile:
-
+You can create a secret with certificate in a different way, for example using oc/kubectl client
 ----
```

Comment:
I don't understand your comment. @rkratky can you elaborate?

Reply:
The comment is somehow misformatted. Not sure how it happened. I meant to suggest the following (it allows formatting to happen in the code block):
```suggestion
[subs="+quotes"]
----
```

Reply:
I think I still do not understand what's meant.  Can you give an example that I can just copy and paste?

Reply:
I used the 'suggested change' mechanism. It should be possible to just click 'accept suggestion', and it will be added as a commit. Anyway, here's the entire code block:

```
[subs="+quotes"]
----
$ oc create secret generic self-signed-certificate --from-file=__</path/to/certificate>__
----

Reply:
got it. fixed. thanks.